README.md

```diff
@@ -82,26 +69,25 @@ To get the name of the device enter:
     emulator -list-avds
 
 Then open one terminal and run the command:
-    react-native start
-
-This terminal will prevent an error on run.
+    
+    `npm start`
 
 Then open another terminal and run:
-    react-native run-android
+
+    `npm run android`
 
 This may take a while especially the first time.
 If the app doesn't open automatically go to apps on the device and click AlumniApp.
-You should see a welcome message.
 
 ## Common errors
 
 ### No tools.jar
-Linux distros often use open java sdk which react does not like and it will complain
```

Comment:
This is related purely to an improperly set up system environment. Which happens if installs didn't go exactly as planned. All that is necessary is that *some* JDK is installed and the proper variables are set if not done automatically. 

Tested with `OpenJDK`, an oracle JDK, Zulu `OpenJDK` variant for Windows and the JetBrains JDK variant.